src/renderer/src/routes/modals/SellTokensDeposit/SellTokensDepositModal.tsx

```diff
@@ -183,11 +185,11 @@ const SellTokensDepositModal = () => {
 
     const transferParams = getServiceTransferParams()
 
-    if (!transferParams) return
-
     const { address, amount, token, account } = actionData
 
-    if (account?.type === 'hardware') {
+    if (!transferParams || !account) return
```

Comment:
Not necessary, just:
```suggestion
    if (!transferParams) return
```